.gitignore

```
@@ -0,0 +1,2 @@
+settings.yaml
+last_id.txt
```

add new line at the end of file
```suggestion
last_id.txt

```